src/mrnet/core/reactions.py

```diff
@@ -86,17 +87,19 @@ def __init__(
                 reactants, products, self.transition_state
             )
 
-        self.reactant_ids = [e.entry_id for e in self.reactants]
-        self.product_ids = [e.entry_id for e in self.products]
-        self.entry_ids = {e.entry_id for e in self.reactants + self.products}
+        self.reactant_eids = np.array([e.entry_id for e in reactants])
+        self.product_eids = np.array([e.entry_id for e in products])
```

Comment:
Originally, I believe it was of the form entry_ids, but I suppose there's no real reason to specify it's an "entry" id, since there isn't really another kind.